index.html

```diff
@@ -155,7 +155,7 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
             <div class="row">
                 <div class="col-md-3 col-sm-6 col-lg-3 col-12">
                     <div class="">
-                        <a href="index.html"><img src="./images/icon.png" alt="" class="img-fluid mb-lg-4" width="100px" height="100px"> </a>
+                        <a href=""><img src="./images/icon.png" alt="" class="img-fluid mb-lg-4" width="100px" height="100px"> </a>
```

Comment:
No need of an anchor tag, with blank anchor tag like this if we click on it the page will reload which is unnecessary.

Reply:
done